pom.xml

```diff
@@ -46,7 +46,7 @@
 		<jaxb20.version>2.0.5</jaxb20.version>
 		<jaxb21.version>2.1.13</jaxb21.version>
 		<jaxb22.version>2.2.11</jaxb22.version>
-		<jaxb23.version>2.3.1</jaxb23.version>
+		<jaxb23.version>2.3.4</jaxb23.version>
```

Comment:
By now version [2.3.5](https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-runtime/2.3.5/) is out.